src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1715,6 +1719,73 @@ private void SetPeriodicEvaluation(WorkspaceModel ws)
             return true;
         }
 
+        // Attempts to reload all the dummy nodes in the current workspace and replaces them with resolved version. 
+        private void ReloadDummyNodes()
+        {
+            JObject dummyNodeJSON = null;
+            WorkspaceModel currentWorkspace = this.CurrentWorkspace;
+
+            if (currentWorkspace == null || string.IsNullOrEmpty(currentWorkspace.FileName))
+            {
+                return;
+            }
+
+            String filePath = Path.GetFullPath(currentWorkspace.FileName);
```

Comment:
I don't think `filePath` is used anywhere.

Reply:
Yes, removing this.